### ISSUE TITLE
Bad Accept-Language header value

### ISSUE THREAD

**Issue by namedgraph** [AtomGraph/LinkedDataHub#10]
Bad `Accept-Language` header value: `en-GB,en;q=0.9,en-US;q=0.8,nl;q=0.7,es-419;q=0.6,es;q=0.5,de;q=0.4`

Browser: Chrome on MacOS
Reported by @warpr

**Comment by namedgraph** [comment]
Reported by @lisp:
```there is some issue with accept-language headers:

berlintoo:src jamesanderson$ curl -v -k https://linkeddatahub.com:4443  -H "Accept-Language: en-US,en;q=0.9,de;q=0.8,es-419;q=0.7,es;q=0.6"
* Rebuilt URL to: https://linkeddatahub.com:4443/
*   Trying 94.130.104.101...
* TCP_NODELAY set
* Connected to linkeddatahub.com (94.130.104.101) port 4443 (#0)
* ALPN, offering http/1.1
* Cipher selection: ALL:!EXPORT:!EXPORT40:!EXPORT56:!aNULL:!LOW:!RC4:@STRENGTH
* successfully set certificate verify locations:
*   CAfile: /anaconda2/ssl/cacert.pem
  CApath: none
* TLSv1.2 (OUT), TLS header, Certificate Status (22):
* TLSv1.2 (OUT), TLS handshake, Client hello (1):
* TLSv1.2 (IN), TLS handshake, Server hello (2):
* TLSv1.2 (IN), TLS handshake, Certificate (11):
* TLSv1.2 (IN), TLS handshake, Server key exchange (12):
* TLSv1.2 (IN), TLS handshake, Request CERT (13):
* TLSv1.2 (IN), TLS handshake, Server finished (14):
* TLSv1.2 (OUT), TLS handshake, Certificate (11):
* TLSv1.2 (OUT), TLS handshake, Client key exchange (16):
* TLSv1.2 (OUT), TLS change cipher, Client hello (1):
* TLSv1.2 (OUT), TLS handshake, Finished (20):
* TLSv1.2 (IN), TLS change cipher, Client hello (1):
* TLSv1.2 (IN), TLS handshake, Finished (20):
* SSL connection using TLSv1.2 / ECDHE-RSA-AES256-GCM-SHA384
* ALPN, server did not agree to a protocol
* Server certificate:
*  subject: CN=linkeddatahub.com
*  start date: Sep 13 21:32:08 2018 GMT
*  expire date: Dec 12 21:32:08 2018 GMT
*  issuer: C=US; O=Let's Encrypt; CN=Let's Encrypt Authority X3
*  SSL certificate verify ok.
> GET / HTTP/1.1
> Host: linkeddatahub.com:4443
> User-Agent: curl/7.60.0
> Accept: */*
> Accept-Language: en-US,en;q=0.9,de;q=0.8,es-419;q=0.7,es;q=0.6
> 
< HTTP/1.1 400 Bad Request
< Server: Apache-Coyote/1.1
< Strict-Transport-Security: max-age=31536000;includeSubDomains
< Content-Type: text/plain
< Transfer-Encoding: chunked
< Date: Sat, 20 Oct 2018 14:43:02 GMT
< Connection: close
< 
* Closing connection 0
* TLSv1.2 (OUT), TLS alert, Client hello (1):
Bad Accept-Language header value: 'en-US,en;q=0.9,de;q=0.8,es-419;q=0.7,es;q=0.6'berlintoo:src jamesanderson$```

**Comment by namedgraph** [comment]
@warpr @lisp looks like `es-419` cannot be parsed a bug in Jersey 1.x.
`JERSEY-2124` issue is resolved in 2.5: https://jersey.github.io/release-notes/2.5.html
We are hoping to upgrade at some point in the near future.

**Comment by namedgraph** [comment]
Should be fixed by the upgrade to Jersey 2.30.1
